saleor/graphql/product/types/products.py

```diff
@@ -188,7 +188,8 @@ class ProductVariant(CountableDjangoObjectType):
         required=True,
         description="Quantity of a product available for sale.",
         deprecation_reason=(
-            "Use the stock field instead. This field will be removed after 2020-07-31."
+            "Use the quantity_available field instead. "
```

Comment:
This should refer to the name in the schema which is camel case:

```suggestion
            "Use the quantityAvailable field instead. "
```